.github/workflows/m68k.yml

```diff
@@ -88,6 +91,19 @@ jobs:
         sudo mount debian-m68k.img vm
         sudo cp $(which qemu-m68k-static) vm/usr/bin/
 
+    - name: Prepare for build with target as JSON spec
+      run: |
+        ./y.sh prepare --only-libcore --cross
+
+    - name: Build sysroot with target as JSON spec
+      run: |
+        ./y.sh build --build-sysroot --target-triple m68k-unknown-linux-gnu --target ${{ github.workspace }}/target_specs/m68k-unknown-linux-gnu.json
```

Comment:
`--build-sysroot` is not an argument to `build` and this `build` command automatically builds the sysroot for now, even though [it's going to change soon](https://github.com/antoyo/rustc_codegen_gcc/pull/9).

Reply:
Fixed, that was just for testing.